babel.config.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-env node */
```

Comment:
Im confused, this file existed before right? Is there a reason this is different from our babelrc?

Reply:
This is a new file for space-kit. We had to do this previously in engine-frontend.